File: packages/java/ygo74-agent-runtime/src/main/java/com/ygo74/agentruntime/routing/RoutingErrors.java
package com.ygo74.agentruntime.routing;

import java.util.Map;

public final class RoutingErrors {
    private RoutingErrors() {}

    public static Map<String, String> routeNotRegistered(String routeKey) {
        return Map.of(
                "code", "route_not_registered",
                "category", "routing",
                "message", "No handler registered for route '" + routeKey + "'");
    }
}

File: packages/dotnet/Ygo74.AgentRuntime/Routing/RoutingErrors.cs
namespace Ygo74.AgentRuntime.Routing;

public static class RoutingErrors
{
    public static object RouteNotRegistered(string routeKey) => new
    {
        code = "route_not_registered",
        category = "routing",
        message = $"No handler registered for route '{routeKey}'"
    };
}
